### ISSUE TITLE
Why CormasModel>>initCells: is in deprecated protocol?

### ISSUE THREAD

**Issue by SergeStinckwich** [cormas/cormas#152]
Why ``CormasModel>>initCells:`` is in deprecated protocol? Same for ``initCells:withArguments:``
Mistake?

**Comment by pierbo** [comment]
```
initCells: initSelector withArguments: argumentCollection
	"The method initSelector <Symbol> with the arguments : argumentCollection, is send to each Cell instance"
```
The first versions of Cormas used to use this method, but currently few models use it actually, as it can be easily replaced by:`
	self theCells do:[:c| c init].  "or a specific #init method of the Cell class"`

**Comment by SergeStinckwich** [comment]
Conway, Diffuse and FireAutomata models use this methods. I'm not sure to understand. You propose to replace:

``self initCells: #init``

by

``self theCells do:[:c| c init1]``

do not seems simpler for me 😄 

BTW, you could wrote:
``self theCells do: #init``

What do you think about last one?

**Comment by SergeStinckwich** [comment]
``initCells:withArguments:`` is also used by the FireAutomata model. What do we do with such methods?

**Comment by ElCep** [comment]
When @pierbo said 
> but currently few models use it actually
It's in the VW cormas. I think it needs to be unified in the current version. 
`self theCells do:[:c| c init1]` is not simpler but it's more explicit. So maybe it's easier for beginners?

`self initCells: #init` can be nice, but we have to document it in the booklet otherwise, just by reading it (if you're not a computer scientist / smalltallk lover) it's not necessarily clear the #simbols

**Comment by pierbo** [comment]
For several years, we tried to simplify the syntax because beginner modellers sometimes had difficulties with the loops (`do:[:c| c xx]` is not obvious). So we looked for ways to avoid this, such as #initCells.
But with experience, I think we made a mistake: modellers must necessarily learn to loop when manipulating lists of agents. The cost of learning is a little bit higher, but it is essential to know how to manipulate collections to become more autonomous.
So, rather than proposing initCells-type methods, I think it's better (and more explicit) to let the modeller write this:
`self theCells do:[:c| c init]`
Moreover, thanks to Pharo, there is also the possibility to write
`self theCells do: #init`. (which looks like Netlogo!)
even if that makes me think that this way of doing might be confusing for beginners: why using blocks and sometimes symbols... but that's another debate.
So, yes for deleting `initCells: #symbol` (or keeping it as deprecated)
